ts/features/itwallet/identification/screens/ItwIdentificationModeSelectionScreen.tsx

```diff
@@ -17,6 +17,13 @@ import {
 import { itwDisabledIdentificationMethodsSelector } from "../../../../store/reducers/backendStatus/remoteConfig";
 import { IOScrollViewWithLargeHeader } from "../../../../components/ui/IOScrollViewWithLargeHeader";
 import { isCIEAuthenticationSupportedSelector } from "../../machine/eid/selectors";
+import { ITW_ROUTES } from "../../navigation/routes";
+
+export type ItwIdentificationModeSelectionScreenNavigationParams =
+  | {
+      eidReissuing?: boolean;
+    }
+  | undefined;
```

Comment:
```suggestion
export type ItwIdentificationModeSelectionScreenNavigationParams = {
      eidReissuing?: boolean;
    };
    
export type ScreenProps = IOStackNavigationRouteProps<
  ItwParamsList,
  "ITW_IDENTIFICATION_MODE_SELECTION"
>

```

I believe this approach is more appropriate because:
- the optional `eidReissuing` parameter is not hidden. This makes it clear to anyone using this screen that the parameter exists and is optional
- by defining a dedicated type for the screen props we are improving readability and maintainability

What do you think?

Reply:
Yes, you're right! Please check here [2ca518f](https://github.com/pagopa/io-app/pull/6580/commits/2ca518f4bc988398916844ffb98a03c95fb143cf) ( `console.log` deleted here [b72bffd](https://github.com/pagopa/io-app/pull/6580/commits/b72bffd9e31c9bdfe8474870a14ce648a7cf2595))